protocol/x/clob/keeper/process_operations.go

```diff
@@ -651,29 +651,41 @@ func (k Keeper) PersistMatchDeleveragingToState(
 			perpetualId,
 		)
 	}
-	deltaQuantumsIsNegative := position.GetIsLong()
+	deltaBaseQuantumsIsNegative := position.GetIsLong()
 
 	for _, fill := range matchDeleveraging.GetFills() {
-		deltaQuantums := new(big.Int).SetUint64(fill.FillAmount)
-		if deltaQuantumsIsNegative {
-			deltaQuantums.Neg(deltaQuantums)
+		deltaBaseQuantums := new(big.Int).SetUint64(fill.FillAmount)
+		if deltaBaseQuantumsIsNegative {
+			deltaBaseQuantums.Neg(deltaBaseQuantums)
+		}
+
+		deltaQuoteQuantums, err := k.getDeleveragingQuoteQuantumsDelta(
```

Comment:
this is where we call ProcessDeleveraging in DeliverTx, we call the helper function to get the delta quote quantums to pass in